modules/tooltip/react/lib/Tooltip.tsx

```diff
@@ -76,6 +76,9 @@ export const Tooltip = ({
       {React.cloneElement(children, {
         ...targetProps,
         ...mergeCallbacks(children.props, targetProps),
+        ...(type === 'muted' && children.props['aria-label']
+          ? {'aria-label': children.props['aria-label']}
+          : {}),
```

Comment:
👍🏻

Reply:
Definitely messy, but this should hold over v4. Things will change in future versions. I'm not sure there will be a "muted" type, but a muted pattern.